src/modifiers/mod_flatten_inheritance.py

```diff
@@ -27,13 +27,23 @@ def apply(dom_root):
         if parent_struct is None:
             raise Exception("Failed to find parent class " + parent_struct.name + " for type " + struct.name)
         
-        parent_fields = parent_struct.clone().list_directly_contained_children_of_type(code_dom.DOMFieldDeclaration)
+        # Create a field with the parent type
+        parent_struct = parent_struct.clone()
+
+        dummy_field = code_dom.DOMFieldDeclaration()
```

Comment:
I'm not sure about this change, TBH... it kinda makes sense for certain types of bindings, but it feels pretty unnatural for "native" C code to have to refer to base class members via a `Parent_<something>` reference rather than treating them as though they are members of the derived class.

If you feel this is useful then I'm more than happy to support it as an option, but I feel like unless there's a clear reason not to the default behaviour should be to flatten the fields into the derived class the way the previous version of this code did, as that's probably the "least surprising" behavior for a lot of users.

Reply:
The original implementation did in fact just append fields:
[`8c2c47f` (#47)](https://github.com/dearimgui/dear_bindings/pull/47/commits/8c2c47f62bc4ad6b185e3c9148c8a8d92a387a43)

I changed it to `Parent_<something>` because I thought it's more explicit and the code is more concise, so no real strong reasoning here. I don't mind reverting it to just appending the parent field.

Reply:
Cool, in that case I think just appending fields is probably the way to go.